.github/workflows/spack.yml

```diff
@@ -48,52 +48,160 @@ jobs:
       matrix:
-        compiler: [gcc] #, clang, intel]
-        gpu: [none] #, cuda, rocm]
-        blas: [openblas] # ,intel-oneapi-mkl]
-        spack_version: [develop] # ,v0.23.1]
-        mpi: [openmpi] #, mpich, mvapich]
+        include:  # Pairwise testing (from generate_test_matrix.py)
+        - arch: x86
+          compiler: gcc
+          mpi: openmpi
+          math-libs: openblas
+          shared: +shared
+          int: +int64
+          openmp: ~openmp
+          eigensolver: +arpack
+          solver: +mumps
+          cuda: ~cuda
 
-    runs-on: palace_ubuntu-latest_16-core
+        # Fails configuring PETSc
+        # - arch: x86
+        #   compiler: llvm
+        #   mpi: mpich
+        #   math-libs: amdblis
+        #   shared: ~shared
+        #   int: ~int64
+        #   openmp: +openmp
+        #   eigensolver: +slepc
+        #   solver: +superlu-dist
+        #   cuda: ~cuda
+
+        # Fails building MFEM with `FC` not found
+        # - arch: arm
+        #   compiler: llvm
+        #   mpi: openmpi
+        #   math-libs: openblas
+        #   shared: ~shared
+        #   int: ~int64
+        #   openmp: ~openmp
+        #   eigensolver: +slepc
+        #   solver: +strumpack
+        #   cuda: ~cuda
+
+        - arch: arm
+          compiler: gcc
+          mpi: mpich
+          math-libs: armpl-gcc
+          shared: +shared
+          int: +int64
+          openmp: +openmp
+          eigensolver: +arpack
+          solver: +strumpack
+          cuda: ~cuda
+
+        # Fails building libxsmm with ifx error None not found
+        # - arch: x86
+        #   compiler: intel-oneapi-compilers
+        #   mpi: intel-oneapi-mpi
+        #   math-libs: intel-oneapi-mkl
+        #   shared: +shared
+        #   int: +int64
+        #   openmp: ~openmp
+        #   eigensolver: +slepc
+        #   solver: +superlu-dist
+        #   cuda: ~cuda
+
+        # Fails building libxsmm with ifx error None not found
+        # - arch: x86
+        #   compiler: intel-oneapi-compilers
+        #   mpi: intel-oneapi-mpi
+        #   math-libs: intel-oneapi-mkl
+        #   shared: ~shared
+        #   int: ~int64
+        #   openmp: +openmp
+        #   eigensolver: +arpack
+        #   solver: +mumps
+        #   cuda: ~cuda
+
+        - arch: x86
+          compiler: llvm
+          mpi: mpich
+          math-libs: amdblis
+          shared: +shared
+          int: +int64
+          openmp: ~openmp
+          eigensolver: +arpack
+          solver: +mumps
+          cuda: ~cuda
+
+        # Fails building superlu with pointer conversion error int vs int_t
+        # - arch: x86
+        #   compiler: gcc
+        #   mpi: openmpi
+        #   math-libs: amdblis
+        #   shared: ~shared
+        #   int: +int64
+        #   openmp: +openmp
+        #   eigensolver: +arpack
+        #   solver: +superlu-dist
+        #   cuda: ~cuda
+
+        # Fails with undefined reference to symbol 'sqrt@@GLIBC_2.17' in libmetis
```

Comment:
Should this be a bug report? Sounds like a math library linkage issue...

Reply:
Same comment as above on good bug reports.